assembl/alembic/versions/053f788ca313_resource_center.py

```diff
@@ -0,0 +1,78 @@
+"""resource_center
+
+Revision ID: 053f788ca313
+Revises: 025d088eb45a
+Create Date: 2017-10-16 18:16:11.329055
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '053f788ca313'
+down_revision = '025d088eb45a'
+
+from alembic import context, op
+import sqlalchemy as sa
+import transaction
+
+from assembl.lib import config
+
+
+def upgrade(pyramid_env):
+    with context.begin_transaction():
+        op.add_column(
+            'discussion', sa.Column('resources_center_title_id',
+            sa.Integer(), sa.ForeignKey('langstring.id')))
+
+        op.create_table(
+            'resource',
+            sa.Column('id', sa.Integer, primary_key=True),
+            sa.Column('type', sa.String(60), nullable=False),
+            sa.Column('discussion_id',
+                sa.Integer,
+                sa.ForeignKey(
+                  'discussion.id',
+                   ondelete="CASCADE",
+                   onupdate="CASCADE"), nullable=False, index=False),
+            sa.Column("title_id", sa.Integer, sa.ForeignKey("langstring.id")),
+            sa.Column("text_id", sa.Integer, sa.ForeignKey("langstring.id")),
+            sa.Column('embed_code', sa.Text()),
+            sa.Column('tombstone_date', sa.DateTime, server_default=None),
```

Comment:
@cedricmessiant  tombstone_date is not needed anymore I think.

Reply:
done